src/math/tiny/tiny_algebra.hpp

```diff
@@ -520,10 +518,10 @@ struct TinyAlgebra {
         tds::where_lt(dd, ss, q2.getY(), -q2.getY()),
         tds::where_lt(dd, ss, q2.getZ(), -q2.getZ()),
         tds::where_lt(dd, ss, q2.getW(), -q2.getW()));
+    closest_end.normalize();
```

Comment:
we shouldn't call Eigen.normalize(), since it does a comparison >0, can you use the Algebra::normalize
 instead?